wp-moj-elasticsearch.php

```diff
@@ -11,6 +11,11 @@
  *
  **/
 
+/**
+ * Disable ElasticPress dashboard sync
+ */
+define('EP_DASHBOARD_SYNC', false);
```

Comment:
Hey, just a quick question. Why are we disabling dashboard sync? was this causing an error?

Reply:
This was also removed. Dashboard sync isn't compatible with our plugin

Reply:
Ok great thanks

Reply:
This was one of the first things that went on the EP side of things. Because it is very slow and cumbersome. We had a dev conversation about it in the early stages of plugin build. We use the CLI exclusively for its speed and detachability.